src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --main-color: #00acdc;
+}
+
+.logo_img {
+  display: block;
+}
+
+.nav_link {
+  text-decoration: none;
+  position: relative;
+  color: black;
+  text-transform: uppercase;
+  height: 15px;
```

Comment:
It's not a good idea to fix the height of the element unless there is a specific requirement for it. It can lead to overflow issues if the content is larger than the specified height.